nginx/config.go

```diff
@@ -186,6 +186,9 @@ http {
 		location / {
 			return 404;
 		}
+		location /nginx_status {
+			stub_status on;
+		}
```

Comment:
I'll read up on whitelisting in the router, pushed a single commit for the configurability